homeassistant/components/broadlink/updater.py

```diff
@@ -18,6 +21,9 @@
 
 def get_update_manager(device):
     """Return an update manager for a given Broadlink device."""
+    if device.api.model.startswith("RM mini"):
+        return BroadlinkRMMini3UpdateManager(device)
```

Comment:
This will do the trick for now, but we should consider segregating interfaces at the library in the future.